integr8ly/integreatly-operator
# MGDAPI-4617 - linter: SA1019 - refactor to remove using deprecated functions

# Issue link

https://issues.redhat.com/browse/MGDAPI-4617

# What

Fix linter staticcheck SA1019 warnings around using deprecated functions

# Verification steps



* Prow checks are successfull
* Install RHOAM from this branch
```
make cluster/prepare/local
make code/run
```
* Verify installation successfull
* Verify rate limiting still works
  * Sign in as admin to 3scale console using credentials in `system-seed` secret in the 3scale namespace
  * Complete the echo api tutorial
  * Note the stage api endpoint in products -> integration -> configuration -> staging apicast
  * Send a load to this endpoint and verify rate limiting works
     * https://github.com/rakyll/hey
  ```
   hey -n 75 -c 1 "<endpoint>"
  ```
  
![image](https://user-images.githubusercontent.com/24636860/212874171-71ed2a2f-7405-4809-a946-6ae027acc334.png)
